test/built-ins/Function/prototype/toString/AsyncGenerator.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: pending
+description: >
+  Function.prototype.toString on an async generator created with the
+  AsyncGenerator constructor.
+features: [async-iteration]
+---*/
```

Comment:
@anba would you mind to add an `info` tag with the respective spec reference?

This is helping us a lot to review and even with information after they land in the code base.